mariner/file_formats/tests/test_cbddlp.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import io
+import pathlib
+from unittest import TestCase
+
+import png
+from pyexpect import expect
+
+from mariner.file_formats.ctb import CTBFile
+
+
+class CTBFileTest(TestCase):
+    def test_loading_ctb_file(self) -> None:
+        path = pathlib.Path(__file__).parent.absolute() / "pyramid.cbddlp"
```

Comment:
is the `.stl` on this file created by you?

Reply:
yes, I can provide it as well if needed - just a pyramid mesh object generated with Blender

Reply:
nah, it's fine! just wanted to confirm this was from a file you produced before we merge this into master